socs/agents/acu/avoidance.py

```diff
@@ -332,26 +394,57 @@ def get_sun_pos(self, az=None, el=None, t=None):
         """
         if t is None:
             t = self._now()
-        v = self._sun(t)
-        qsun = quat.rotation_lonlat(v.ra, v.dec)
+        eff_t = t + self.sun_time_shift
 
-        qzen = coords.CelestialSightLine.naive_az_el(t, 0, np.pi / 2).Q
-        neg_zen_az, zen_el, _ = quat.decompose_lonlat(~qzen * qsun)
+        v = self._sun(eff_t)  # This applies sun_time_shift, internally
```

Comment:
This comment seems inaccurate now, given `+ self.sun_time_shift` was dropped from within `self._sun()` above. It's now added in `eff_t` on line 397.